### ISSUE TITLE
Tune down supybot chattyness

### ISSUE THREAD

**Issue by hpk42** [autocrypt/autocrypt#330]
``_supybot-github`` is showing all github events which appears a bit much. 

I looked around the ``/etc/ghbot`` configuration but could not find config options for filtering events for display.  ``/home/ghbot/autocrypt-github-bot/plugins/Github/config.py`` defines a lot of things, but not sure how to change settings --- by sending commands to the bot, maybe?  @gfa, maybe you can help to shed some light? 

i think we want to restrict events to:

- new PRs, merged PRs against master 
- commits to master 
- open and close issues

Travis does not need to show up and comments to neither PRs nor issues need to show up IMO.  

Bonus: is there a way to condense PR merges? (i can live with it being a bit wordy, doesn't happen so often)

**Comment by hpk42** [comment]
i stopped the bot for now -- it is really is too spammy and disrupts human communication too much.  I think it should be further tested on a separate channel before activating it on #autocrypt again.

**Comment by gfa** [comment]
GH wekbooks are not so granular, the closest mapping would be

```
Issues

Issue opened, edited, closed, reopened, assigned, unassigned, labeled, unlabeled, milestoned, or demilestoned. 

Pull request
Pull request opened, closed, reopened, edited, assigned, unassigned, review requested, review request removed, labeled, unlabeled, or synchronized. 
```

**Comment by gfa** [comment]
I've enabled the bot again, lets see how it works for a few days. if ppl feels that is still to much, we shut it down again

**Comment by gfa** [comment]
For the record, GH was sending _all_ events to the bot

**Comment by hpk42** [comment]
seems good now.  didn't know you can set fine-grained event filtering on the github side. good!
